src/App.js

```diff
@@ -77,6 +78,11 @@ class App extends Component {
 
             <Route exact path="/browse" render={() => <BrowseItems />} />
 
+
+            <Route exact path="/browse/:productID" render={(routeProps) => {
+              console.log(routeProps);
```

Comment:
Probably don't need to log this here. Can we comment it out?

Reply:
OK.